core/src/main/java/com/amplifyframework/Platform.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework;
+
+/**
+ * Enum to represent various platforms that use Amplify library for
+ * tracking usage metrics.
+ */
+public enum Platform {
+    /**
+     * Represents the Android platform.
+     */
```

Comment:
This enum should not be used actually haha. It is only used internally by `UserAgent` to construct the default android portion. I should actually document that a user should NOT use this, since it's always accounted for already.